development/tools/jobs/kyma/kyma_integration_test.go

```diff
@@ -305,57 +300,50 @@ func TestKymaGKECentralConnectorJobsPresubmit(t *testing.T) {
 	assert.Equal(t, 10, actualJob.MaxConcurrency)
 	tester.AssertThatHasExtraRefTestInfra(t, actualJob.JobBase.UtilityConfig, "master")
 	tester.AssertThatSpecifiesResourceRequests(t, actualJob.JobBase)
-	assert.Equal(t, tester.ImageBootstrapHelm20181121, actualJob.Spec.Containers[0].Image)
+	assert.Equal(t, tester.ImageKymaClusterInfra20190528, actualJob.Spec.Containers[0].Image)
 	tester.AssertThatHasPresets(t, actualJob.JobBase, tester.PresetGCProjectEnv, tester.PresetBuildPr,
-		tester.PresetDindEnabled, "preset-sa-gke-kyma-integration",
+		tester.PresetDindEnabled, tester.PresetKymaGuardBotGithubToken, "preset-sa-gke-kyma-integration",
 		"preset-gc-compute-envs", "preset-docker-push-repository-gke-integration")
 }
 
 func TestKymaIntegrationJobsPostsubmit(t *testing.T) {
 	tests := map[string]struct {
 		givenJobName string
 		expPresets   []tester.Preset
-		expJobImage  string
 	}{
 		"Should contains the kyma-integration job": {
 			givenJobName: "post-master-kyma-integration",
 
 			expPresets: []tester.Preset{
-				tester.PresetGCProjectEnv, "preset-sa-vm-kyma-integration",
+				tester.PresetGCProjectEnv, tester.PresetKymaGuardBotGithubToken, "preset-sa-vm-kyma-integration",
 			},
-
-			expJobImage: tester.ImageBootstrap001,
 		},
 		"Should contains the gke-integration job": {
 			givenJobName: "post-master-kyma-gke-integration",
 
 			expPresets: []tester.Preset{
 				tester.PresetGCProjectEnv, tester.PresetBuildMaster,
-				tester.PresetDindEnabled, "preset-sa-gke-kyma-integration",
+				tester.PresetDindEnabled, tester.PresetKymaGuardBotGithubToken, "preset-sa-gke-kyma-integration",
```

Comment:
what is the purpose of preset: `preset-kyma-guard-bot-github-identity` ?

Reply:
I just followed the previous approach. Right now it's not used anywhere but at least in context of the `GithubToken` you have information about its identity. Should be removed?